### PR TITLE
Dynamic loader for injecting into other websites

### PR DESCRIPTION
**Pull request by jamescw**

### CODE REVIEW

File: transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py [hunk 1]
```diff
@@ -990,6 +990,20 @@ def create_labels(row):
...
+    on="Source_Id",
+    how="left",
+    sort=False,
+)
+# assign source link for TMEE, url: UNICEF_RDM/indicator_code
+tmee_source_link = df_sources.Source_Link.isnull()
+unicef_rdm_url = "https://data.unicef.org/indicator-profile/{helix_code}/"
+df_sources.loc[tmee_source_link, "Source_Link"] = df_sources[
+    tmee_source_link
+].Code.apply(lambda x: unicef_rdm_url.format(helix_code=x))
```
Comment (jamescw) [inline transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py:1006]: @beto-Sibileau and @lbassil can you check this still looks good after the merge?

Comment (lbassil) [reply transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py:1006]: It LGTM... not sure what exactly needs to be checked, I did a general checking in addition to the source link and it appears working normally. 

DO we need to check the newly added html pages?

Comment (lbassil) [reply transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py:1006]: image

@beto-Sibileau , I only noticed in the adolescent cross sectoral page, under area 6: Participation and Civil Rights, that indicators starting with Adolescents (shown in the above figure) have empty source. Should we exclude them from the dashboard? We surely need to review the indicators under the cross sectoral pages too.

Comment (beto-Sibileau) [reply transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py:1006]: Hi guys! I´m sorry for engaging late on this one, I couldn´t join before!

In particular to your question @lbassil , I understand the reason for those indicators you mentioned (*Participation and Civil Rights, starting with Adolescents*), not to have a source, is the missmatch between Subdomain names.

Note in the dashboard config you entered **Education, Leisure, and Culture** and in the data dictionary it is under **Leisure, and Culture** only. I took the latter from the latest review you shared at the beginning of the year:

image

I totally agree that eventually these details and in particular cross sectoral pages config must be reviewed! Thanks =)

**Pull request merged**